templates/login.html

```diff
@@ -0,0 +1,18 @@
+<html>
```

Comment:
Templates should extend the base template. Also, please make sure to sync your fork and place templates in the `web/templates/pages` folder.